jelastic/api/utils/__init__.py

```diff
@@ -115,6 +115,7 @@ def CreateEnvTask(
         self,
         script: str,
         trigger: dict[TRIGGER_TYPE, str | int],
+        env_name: list[str] = None,
```

Comment:
1. This is a string
2. It's not optioanl although the docs do suggest that

If I dont pass the `env_name`, an error is raised

```
invalid parameter [envName] for method [GetEnvInfo] in service [ControlService]
```

Reply:
but sir in docs this is optional field